.gitignore

```diff
@@ -139,7 +140,6 @@ packages/fetchai/connections/p2p_libp2p/libp2p_node/libp2p_node
 !plugins/aea-ledger-fetchai/tests/data/dummy_contract/build
 
 deploy-image/.env
-deploy-image/packages/valory
```

Comment:
why the change?

Reply:
We don't copy packages to the image directory anymore